### ISSUE TITLE
Design system docs - Colours reserved for accessibility missing from Colour page

### ISSUE THREAD

**Issue by KateMacdonald** [WestpacGEL/GEL-next#578]
I have re-added the correct copy for this in my branch, however it doens't let me add the colour swatches.

Original GEL
![image](https://github.com/WestpacGEL/GEL-next/assets/6667337/7a4e9c16-602d-4c04-ad92-13b511c5edbd)

GEL4
![image](https://github.com/WestpacGEL/GEL-next/assets/6667337/dc6007f9-ab97-4f68-8c36-967e529ac226)

**Comment by samithaf** [comment]
@KateMacdonald colour component is a custom component specific to GEL Docs and we haven't enable it for Accessibility tab. I will fix that up on Monday.

**Comment by samithaf** [comment]
@KateMacdonald fixed in your branch.

**Comment by KateMacdonald** [comment]
@samithaf When I try to look at the page in the CMS i get this:
![image](https://github.com/WestpacGEL/GEL-next/assets/6667337/485111f0-499c-4d40-b2f1-d4b918563869)
